visualization/tools/librealsense_viewer.cpp

```diff
@@ -98,16 +115,6 @@ class LibRealSenseViewer
     LibRealSenseViewer (pcl::LibRealSenseGrabber& grabber)
     : grabber_ (grabber)
     , viewer_ ("LibRealSense Viewer")
-    , grabber_second_ (grabber)
```

Comment:
No, one grabber correspond with only one device, if the others want to use the same data from one captured device, they have to register themselves to the same grabber. :)